cmake/Buildretargeting.cmake

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2018  iCub Facility, Istituto Italiano di Tecnologia
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+find_or_build_package(YARP QUIET)
+find_or_build_package(ICUB QUIET)
+find_or_build_package(iDynTree QUIET)
+
+ycm_ep_helper(retargeting TYPE GIT
+              STYLE GITHUB
+              REPOSITORY robotology/walking-teleoperation.git
+              TAG master
+              COMPONENT teleoperation
+              FOLDER robotology
+#              CMAKE_ARGS -DWALKING_TELEOPERATION_USES_CYBSDK:BOOL=${ROBOTOLOGY_USES_CYB_SDK}
+#                         -DWALKING_TELEOPERATION_USES_OCULUSSDK:BOOL=${ROBOTOLOGY_USES_OCULUS_SDK}
```

Comment:
Why are this commented?

Reply:
Because currently, we are not using these arguments in the external project. I did not delete them, because maybe in future we may need them!

Reply:
Ah, I got it. Basically we need to have the `WALKING_TELEOPERATION_USES_CYBSDK` to make sure that the teleoperation repo compiles on Windows even without the CybSDK, right? Can you open an issue in the walking-teleoperation repo to track this?

Reply:
Actually, it is different a little bit, we do not need the cybSDK at compile time, because we included them as a part of our module ( https://github.com/robotology/walking-teleoperation/tree/master/modules/Virtualizer_module/CybSDK). While in runtime, we need the cybsdk.dll which exists in cybSDK.